src/applications/vaos/new-appointment/schemas/addressSchema.js

```diff
@@ -0,0 +1,153 @@
+import { validateWhiteSpace } from 'platform/forms/validations';
```

Comment:
Generally we'd call these fields or field definitions, rather than schemas. Similar to here: https://github.com/department-of-veterans-affairs/vets-website/tree/master/src/platform/forms-system/src/js/definitions